reconcile/gitlab_housekeeping.py

```diff
@@ -113,6 +113,18 @@ def rebase_merge_requests(dry_run, gl, rebase_limit):
             if not good_to_rebase:
                 continue
 
+            pipelines = mr.pipelines()
+            if not pipelines:
+                continue
+
+            # posibble statuses:
```

Comment:
type: possible

Reply:
did you meen typo?

Reply:
was that not funny? :) thanks for the catch!